ASScreenRecorder/ASScreenRecorder.m

```diff
@@ -51,8 +49,9 @@ - (instancetype)init
     self = [super init];
     if (self) {
         _videoQuality = ASSScreenRecorderVideoQualityMedium;
```

Comment:
Please store the following singletons as properties, it will make our life easier once we develop tests:
- `[UIApplication sharedApplication]`
- `[UIScreen mainScreen]`
- `[NSFileManager defaultManager]`